packages/ckeditor5-table/src/tableproperties/tablepropertiesediting.ts

```diff
@@ -222,20 +222,22 @@ function enableAlignmentProperty( schema: Schema, conversion: Conversion, defaul
 				key: 'tableAlignment',
 				value: ( viewElement: ViewElement, conversionApi: UpcastConversionApi, data: UpcastConversionData<ViewElement> ) => {
```

Comment:
@mmotyczynska / @niegowski I adjusted default upcasts for `align` property to align description of our task. 

> - Defaults for Layout tables:
>    - Don’t set border on **layout tables** (none).
>    - Don’t set alignment on **layout tables** (none). Reset stylesheets.

I added appreciate tests for that https://github.com/ckeditor/ckeditor5/pull/18213/commits/fa2b57b589d32cc021f1f774a4c3861805e0282b#diff-2ee3d1172377581facb2c9a958a98da3624374db6cbec8d9be4d3777a17fdcf3R1623-R1675

Reply:
Discussed, reverting. I'll leave comment.